### ISSUE TITLE
Error with CP4I : New PVC values contain breaking changes

### ISSUE THREAD

**Issue by abinashkd** [ibm-messaging/mq-container#433]
I am using an OpenShift cluster and provisioning MQ via Cloud Pak dashboard. 
While provision it is getting this below error: "New PVC values contain breaking changes"

```
apiVersion: mq.ibm.com/v1beta1
kind: QueueManager
metadata:
  creationTimestamp: '2020-11-24T13:56:02Z'
  generation: 2
  name: cp4i-ibm-mq-v4
  namespace: cp4i-demo1
  resourceVersion: '726683'
  selfLink: >-
    /apis/mq.ibm.com/v1beta1/namespaces/cp4i-demo1/queuemanagers/cp4i-ibm-mq-v4
  uid: xxx
spec:
  license:
    accept: true
    license: xxxx
    metric: ProcessorValueUnit
    use: NonProduction
  queueManager:
    availability:
      type: MultiInstance
    debug: false
    imagePullPolicy: IfNotPresent
    livenessProbe:
      failureThreshold: 1
      initialDelaySeconds: 90
      periodSeconds: 10
      successThreshold: 1
      timeoutSeconds: 5
    logFormat: Basic
    metrics:
      enabled: true
    name: QUICKSTART
    readinessProbe:
      failureThreshold: 1
      initialDelaySeconds: 10
      periodSeconds: 5
      successThreshold: 1
      timeoutSeconds: 3
    resources:
      limits:
        cpu: '1'
        memory: 1Gi
      requests:
        cpu: '1'
        memory: 1Gi
    storage:
      persistedData:
        class: ibmc-file-gold-gid
        enabled: true
        size: 2Gi
        type: persistent-claim
      queueManager:
        class: ibmc-file-gold-gid
        size: 2Gi
        type: persistent-claim
      recoveryLogs:
        class: ibmc-file-gold-gid
        enabled: true
        size: 2Gi
        type: persistent-claim
  securityContext:
    initVolumeAsRoot: false
    supplementalGroups:
      - 99
  template:
    pod:
      containers:
        - env:
            - name: MQSNOAUT
              value: 'yes'
          name: qmgr
          resources: {}
  terminationGracePeriodSeconds: 30
  tracing:
    agent: {}
    collector: {}
    enabled: false
    namespace: ''
  version: 9.2.0.0-r1
  web:
    enabled: true
status:
  adminUiUrl: >-
    xxxx
  conditions:
    - lastTransitionTime: '2020-11-24T13:56:12Z'
      message: >-
        New PVC values contain breaking changes:
        cp4i-ibm-mq-v4-ibm-mq-persisted-data [cp4i-demo1]
      reason: Failed
      status: 'True'
      type: Error
  endpoints:
    - name: ui
      type: UI
      uri: >-
        xxxx
  name: QUICKSTART
  phase: Failed
  versions:
    available:
      channels: []
      versions:
        - name: 9.2.0.0-r1
    reconciled: 9.2.0.0-r1

```

**Comment by arthurbarr** [comment]
This is probably because you've changed the QueueManager in an invalid way.  We've tightened up the error checking over the last few releases to try and prevent this.  Which version of the MQ Operator are you using?  What was the change you made? (i.e. what did the QueueManager look like before?)

**Comment by arthurbarr** [comment]
It's not valid (for example) to change between single-instance and multi-instance.

**Comment by abinashkd** [comment]
@arthurbarr I am creating fresh Multi instance Queue Manger and not modifying any params, once I click create in cloud pak. 
MQ Operator version: 1.1.0 
Open Shift version: 4.4.29
CloudPak version : Cloud Pak for Integration version 2020.3.1-1

**Comment by sdmarshall79** [comment]
This looks like a known bug that was found with MQ Operator **v1.1.0**.  It occurs when deploying a multi-instance queue manager on IBM Cloud storage.

This was fixed in MQ Operator **v1.2.0** - can you please try upgrading to the latest MQ Operator and the issue should be resolved.

**Comment by abinashkd** [comment]
@sdmarshall79 Indeed upgrading to MQ Operator v1.2.0, resolved the issue. Thank you so much for the help.
